### PR TITLE
Fix/product structured data with blank description

### PR DESCRIPTION
**Pull request by EmCousin**
Book products with a blank description raise a 500 error when navigating to the products page.

### References
- Link to [reproduce in production](https://japanesenotes.gumroad.com/l/aiueo?layout=profile)
- [Bugsnag error](https://app.bugsnag.com/gumroad/web/errors/68ee6e9825e7c232ffb0f742?filters[error.status]=new&filters[event.since]=1d&filters[app.release_stage]=production&sort=events)

### CODE REVIEW

File: app/models/concerns/product/structured_data.rb [hunk 1]
```diff
@@ -39,4 +39,11 @@ def build_book_work_examples
         work_example
       end
     end
+
+    def product_description
+      (custom_summary.presence || strip_tags(html_safe_description).presence)
+        .to_s
+        .truncate(160)
+        .presence
+    end
```
Comment (Hanaffi) [inline app/models/concerns/product/structured_data.rb:48]: Thanks for catching this and putting up a fix! I hadn’t considered that edge case. Let me know if I can help test or refine anything.

Comment (EmCousin) [reply app/models/concerns/product/structured_data.rb:48]: I should have caught that at review time. Don't worry.
